pages/getting-access.md

```diff
@@ -1,15 +1,23 @@
-# Getting access to CCR 
+# Getting access to CCR
 
 CCR offers a variety of resources for researchers to use in their own projects.
-To get started with CCR resources you need the following:
+Regardless of what resources you want to use, you need to start by:
 
-- A CCR user account
-- An active allocation in ColdFront
+- [Creating a CCR user account](#creating-a-ccr-user-account)
+- [Get added to an active allocation for the resource(s) you want to use](#requesting-an-allocation-in-coldfront)
+
+!!! Warning  
+    Access to most CCR resources is restricted to UB and Roswell Park networks (either on campus or connected to their VPN services).  Users without access to UB or RPCI accounts should contact CCR Help for VPN access.
 
 ## Creating a CCR user account
 
-TODO
+Users with UB or Roswell Park email addresses can create themselves a CCR user account following [these directions](../portals/idm/#create-a-ccr-system-account).  Once your account is created, make sure to [follow the instructions for enabling two factor authentication](../portals/idm/#managing-two-factor-authentication).  Now it's time to request access to the resources or services you want to use.
 
 ## Requesting an Allocation in ColdFront
 
-TODO
+ColdFront is CCR's resource and allocation management system.  ColdFront manages all of CCR's resources and the access all users have to those resources.  Faculty members (Principal investigators) must create themselves a project in ColdFront, request allocations for the resources their group wants to use, and then add their group members to their project.  Once an allocation is activated, users on that allocation will be able to access the resource.  [More details about ColdFront can be found here](../portals/coldfront).
```

Comment:
Fix page link, should be: `[More details about ColdFront can be found here](portals/coldfront.md)`